static/elements/chromedash-roadmap-milestone-card.js

```diff
@@ -104,15 +107,20 @@ class ChromedashRoadmapMilestoneCard extends LitElement {
   }
 
   _computeDaysUntil(dateStr) {
+    const date = new Date(dateStr);
+    if (isNaN(date)) {
+      return nothing;
+    }
     const today = new Date();
-    const diff = this._dateDiffInDays(new Date(dateStr), today);
-    const dayWord = diff.days == 1 ? 'day' : 'days';
-
-    if (diff.future) {
-      return `in ${diff.days} ${dayWord}`;
-    } else {
-      return `${diff.days} ${dayWord} ago`;
+    const diff = this._dateDiffInDays(date, today);
+    // If stable is (or was) very recent, we don't want to display relative time
+    // in small increments (e.g. "in 39 minutes"), so we show "coming soon".
+    if (diff.days < 1) {
+      return 'coming soon';
```

Comment:
I think that we do need to show something specific for dates in the future, because people use the site to coordinate their future actions.  If sl-relative-time cannot do future dates, then we could keep the `in N days` logic that was there.  I'm not too concerned about the human-readability of far future dates  (e.g., 'new month' or 'in 5 months') because those milestones will likely be empty anyway.

Reply:
Whoops.  I misunderstood this code.  Your commit comment cleared it up.